README.md

```diff
@@ -36,6 +36,12 @@ The technology stack for this style can be summarized below:
 
 <img src="doc-img/architecture.drawio.svg" alt="Americana technology stack" />
 
+## Data Sources
+
+The OpenStreetMap Americana style is built upon the [OpenMapTiles schema](https://openmaptiles.org/schema/), which includes:
+* Feature data from OpenStreetMap
+* Translated place name labels from [Wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page)
```

Comment:
Is it only places, or anything tagged with `wikidata`?

Reply:
Good point, just grepped the openmaptiles code base and got the full list.

Reply:
Natural Earth?

Reply:
Thanks, just added Natural Earth.  Also I realize I'm missing lake labels.

Reply:
Actually, lake lines are generated from OSM so that's OK.